### PR TITLE
Merge develop into master

### PR DESCRIPTION
**Pull request by jayrbolton**
App tests pass, and manually tested on CI

We could push to the beta version in prod and have any QA folks test it further before releasing.

### CODE REVIEW

File: kbase.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ service-language:
     python
 
 module-version:
-    2.3.1
+    2.4.0
 
 owners:
-    [msneddon, rsutormin, dylan, mclark58]
+    [dylan, jayrbolton, bsadkhin]
```
Comment (bio-boris) [inline kbase.yml]: should we be deleting old owners?

Comment (jayrbolton) [reply kbase.yml]: I'm not really sure why we would keep perms for people not still on the project? I'm assuming they're not going to come back and work on this :P

Comment (bio-boris) [reply kbase.yml]: (I secretly hope they will!) Mostly I thought for giving them credit reasons.

Comment (jayrbolton) [reply kbase.yml]: Oh good point, i wonder what would be the best way to credit authors besides this `owners` field? Display.yaml?

**Comment by jayrbolton** [comment]
(note that tests on travis are failing because appdev and/or CI is down: ` **ERROR** - method-spec validation failed: connect timed out`)

File: Dockerfile [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 # Install MEGAHIT
 WORKDIR /kb/module
-RUN \
-  git clone https://github.com/voutcn/megahit.git && \
-  cd megahit && \
-  git checkout tags/v1.1.1 && \
-  make
+ARG filename=MEGAHIT-1.2.8-Linux-x86_64-static
+RUN cd /opt && \
+  curl -L -O https://github.com/voutcn/megahit/releases/download/v1.2.8/${filename}.tar.gz && \
```
Comment (psdehal) [inline Dockerfile]: The latest version is 1.2.9, it has some fixes, you should update this and update the changelog and display.yaml

Comment (jayrbolton) [reply Dockerfile]: Good catch, that must be recent. I'll update once I have a few mins

**Comment by jayrbolton** [comment]
There is some sdk validation error with the display.yaml. Will fix

File: Dockerfile (hunk 2 above)
Comment (jayrbolton) [reply Dockerfile]: Updated

**Comment by jayrbolton** [comment]
Fixed display.yaml and bumped to 1.2.9

**Pull request merged**